pkg/apis/serving/k8s_validation.go

```diff
@@ -31,12 +31,14 @@ import (
 	"knative.dev/pkg/apis"
 	"knative.dev/pkg/profiling"
 	"knative.dev/serving/pkg/apis/config"
+	"knative.dev/serving/pkg/apis/serving/helpers"
 	"knative.dev/serving/pkg/networking"
 )
 
 const (
-	minUserID, maxUserID   = 0, math.MaxInt32
-	minGroupID, maxGroupID = 0, math.MaxInt32
+	minUserID, maxUserID                     = 0, math.MaxInt32
+	minGroupID, maxGroupID                   = 0, math.MaxInt32
+	PodSpecPersistenceVolumesWriteAnnotation = "features.knative.dev/podspec-persistence-volumes-write"
```

Comment:
I think this was "persistent" above
```suggestion
	PodSpecPersistentVolumesWriteAnnotation = "features.knative.dev/podspec-persistent-volumes-write"
```

Reply:
Correct, will change.